src/test/java/LaunutsApiTest.java

```diff
@@ -123,6 +204,29 @@ public void testCoordinates() throws Exception {
                 coordinatesGot);
     }
 
+    @Test
+    public void testLauCoordinates() throws Exception {
+        String arrays = "[[[49.06881350000003,10.868307500000071],[49.07946209700003,10.874846536000064]," +
+                "[49.08743217800003,10.873127329000056],[49.09703050000007,10.88661650000006]," +
+                "[49.08418420000004,10.907111248000035],[49.09281790500006,10.919888021000077]," +
+                "[49.09022463800005,10.936474906000058],[49.08447845000006,10.941273599000056]," +
+                "[49.08544671300007,10.952453879000075],[49.090076002000046,10.957865948000062]," +
+                "[49.08881412200003,10.975496303000057],[49.08278450000006,10.977152500000045]," +
+                "[49.079923198000074,10.97816343200003],[49.07594637600005,10.98861557500004]," +
+                "[49.062784571000066,10.992995144000076],[49.06030227400004,10.987197021000043]," +
+                "[49.05406831400006,10.985828591000029],[49.04750100000007,10.97670850000003]," +
+                "[49.04533051000004,10.96052885000006],[49.05291532000007,10.951518226000076]," +
+                "[49.05679616900005,10.936995211000067],[49.06139827900006,10.93386840100004]," +
+                "[49.06382272500008,10.919888021000077],[49.06140636200007,10.919888021000077]," +
+                "[49.05922252100004,10.919888021000077],[49.05550200000005,10.899600500000076]," +
+                "[49.05661288600004,10.882821964000073],[49.06403397200006,10.87693803600007]," +
+                "[49.06881350000003,10.868307500000071]]]";
+
+        String coordinatesGot = lauCoordinates.toString();
+        Assert.assertEquals(arrays,
+                coordinatesGot);
+    }
+
```

Comment:
Could you also test the polygon coordinates of TTL response?

Reply:
Done